js/src/views/Contract/contract.js

```diff
@@ -79,9 +83,17 @@ class Contract extends Component {
     }
   }
 
+  componentWillUnmount () {
+    const { api } = this.context;
+    const { subscriptionId, blockSubscriptionId, contract } = this.state;
+
+    api.unsubscribe('eth_blockNumber', blockSubscriptionId);
+    contract.unsubscribe(subscriptionId);
```

Comment:
I agree it would be easier, but the unsubscribe method takes as a first argument the name (and the exposed unsubscribe method didn't, so it wasn't working at that's what I modified).
Should I take care of this in this PR ?